packages/vuetify/src/components/VBottomNavigation/VBottomNavigation.tsx

```diff
@@ -0,0 +1,107 @@
+// Styles
+import './VBottomNavigation.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+import { useBackgroundColor } from '@/composables/color'
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Utilities
+import { computed, defineComponent, toRef } from 'vue'
+import { convertToUnit } from '@/util'
+import { makeProps } from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VBottomNavigation',
+
+  props: makeProps({
+    color: String,
+    flat: Boolean,
```

Comment:
this should be removed

Reply:
Reasoning? Other components already converted support this prop.

Reply:
it has no effect as far as I can see

Reply:
Gotcha, not hooked in. I think I'll remove with the intent to add a **flat** property to elevation in the future.